packages/components/README.md

```diff
@@ -120,6 +120,21 @@ transform: (importName) =>`@redhat-cloud-services/frontend-components/${FECMappe
 transform: (importName) =>`@redhat-cloud-services/frontend-components/esm/${FECMapper[importName] || importName}`,
 
 ```
+## Running with another applicaiton
+
+This set up should work with most applications but individual apps may have dependency conflicts.
+In this example Insights-Inventory-Frontend will be used 
+- In FEC, navigate to packages/components -> here run `npm link`
+- In Inventory root, run `npm link @redhat-cloud-services/frontend-components`
+- Verify it actually linked with  `ls -l node_modules/@redhat-cloud-services/frontend-components`
+- In Inventorys fec.config.js , attach
```

Comment:
@Hyperkid123 Maybe we could add this to the `fec` CLIs config and enable it automagically when it sees any symlink in the `@redhat-cloud-services` packages?